sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Generated/ConversationsModelFactory.cs

```diff
@@ -52,9 +52,15 @@ public static JobStateAutoGenerated JobStateAutoGenerated(string displayName = n
 
         /// <summary> Initializes a new instance of ConversationTasksState. </summary>
         /// <param name="tasks"></param>
+        /// <exception cref="ArgumentNullException"> <paramref name="tasks"/> is null. </exception>
         /// <returns> A new <see cref="Conversations.ConversationTasksState"/> instance for mocking. </returns>
         public static ConversationTasksState ConversationTasksState(ConversationTasksStateTasks tasks = null)
         {
+            if (tasks == null)
```

Comment:
I realize this is already merged, but:

1. Why not just let the constructor throw appropriately rather than duplicating logic?
2. Even if we duplicate logic, if indeed this is a required parameter why do we still declare it as optional?

/cc @annelo-msft @AlexanderSher